code/api/src/migrations/20201208215930-modify_users_add_primary_style.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
           'primaryStyle',
           {
             type: Sequelize.STRING,
-            allowNull: false,
+              allowNull: true,
```

Comment:
Great catch that users needed to allow nulls here

Reply:
Ditto, great realization.